### ISSUE TITLE
Visual integration test errors

### ISSUE THREAD

**Issue by jhsu802701** [octobox/octobox#1162]
So far, I've downloaded the source code and figured out most of the setup needed.  For several of the visual integration tests, I'm getting the following error:
```
Selenium::WebDriver::Error::UnknownError: invalid argument: can't kill an exited
 process
```
What am I missing?

**Comment by andrew** [comment]
That’s a weird one, I’ve not tried the visual tests locally for a couple weeks, will see if I get the same error first thing tomorrow when I’m back at my computer

**Comment by andrew** [comment]
This could be related to the version of geckodriver you have installed https://stackoverflow.com/a/52420400/120434

**Comment by jhsu802701** [comment]
Which version of geckodriver is the correct one?  I'm using v0.22.0 because it's working in the Travis environment.

**Comment by andrew** [comment]
@jhsu802701 I got the same error as you initially, then I noticed that firefox has popped up an update screen, I quit and restarted it and then the tests passed the second time on mac.

**Comment by BenJam** [comment]
the visual tests can be a little squiffy, might be best just to remove them soon.

**Comment by jhsu802701** [comment]
No, but I haven't really had the time to pursue this further since I last posted here.
